packages/terra-worklist-data-grid/src/WorklistDataGrid.module.scss

```diff
@@ -0,0 +1,47 @@
+// Themes
+@import './orion-fusion-theme/WorklistDataGrid.module';
+@import './clinical-lowlight-theme/WorklistDataGrid.module';
+
+:local {
+  .container {
+    height: 100%;
+  }
+
+  [role='grid'] [tabIndex='0']:focus {
```

Comment:
Wouldn't we want to use the worklist data grid class for the selector?

Reply:
Changed to use the class instead of the role selector for the grid.